.eslintrc.js

```diff
@@ -52,28 +52,21 @@ const legacyCode = {
 module.exports = {
   extends: [
     '@wagtail/eslint-config-wagtail',
+    'plugin:import/typescript',
     'plugin:@typescript-eslint/recommended',
   ],
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
+  plugins: ['import', '@typescript-eslint'],
```

Comment:
Are those two really needed here? Since we’re extending from `plugin:import/typescript` and `plugin:@typescript-eslint/recommended` above, I would have expected that to be enough.

Reply:
Ok. It wasn't working for me until I did this but maybe I got this issue confused with the peer dependency issue.

Reply:
thanks, fixed it a better way - had to configure the import resolver
https://stackoverflow.com/questions/55198502/using-eslint-with-typescript-unable-to-resolve-path-to-module